bin.src/makeLsstCameraRepository.py

```diff
@@ -258,6 +259,13 @@ def makeDetectorConfigs(detectorLayoutFile, phosimVersion):
             detectorConfigs.append(detConfig)
     return detectorConfigs
 
+
+def getPhosimVersion(defaultDataDir):
+    """Return the phosim version from data/phosim_version.txt"""
+    with open(os.path.join(defaultDataDir, 'phosim_version.txt')) as infile:
+        return infile.read().strip()
```

Comment:
Since a person has to be involved in the process, I think it's worth having them explicitly put the version number in the file as they go.